src/components/_App/Layout.tsx

```diff
@@ -8,25 +8,37 @@ import Messages from "../../utils/messages";
 import Breadcrumbs from "../Shared/Breadcrumbs";
 import Toast from "../Shared/Toast";
 import muiTheme from "../../styles/Shared/theme";
+import { useEffect, useState } from "react";
 
 interface Props {
   children: React.ReactChild;
 }
 
 const Layout = ({ children }: Props) => {
+  /* TODO: Move this long comment to documentation
+  Makes sure component is completely mounted and all matching html tags are present
+   before adding additional components inside. Otherwise closing tags can be missed
+   and classes incorrectly assigned to wrong HTML elements */
+  const [mounted, setMounted] = useState(false);
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
   return (
-    <ThemeProvider theme={muiTheme}>
-      <Head>
-        <HeadContent />
-        <title>{Messages.brand()}</title>
-      </Head>
-      <Header />
-      <Container maxWidth="sm">
-        <Breadcrumbs />
-        {children}
-        <Toast />
-      </Container>
-    </ThemeProvider>
+    mounted && (
```

Comment:
Let's make this more similar to how `Feed` is currently, to prevent any return type errors in `_app`